pages/linux/kdocker.md

```diff
@@ -0,0 +1,28 @@
+# kdocker
+
+> Easily dock applications to system tray.
+> More information: <https://github.com/user-none/KDocker>.
+
+- Display a cursor to send a window to system tray when pressing the left mouse button (press any other mouse button to cancel):
+
+`kdocker`
+
+- Open an application and send it to system tray:
+
+`kdocker {{application}}`
+
+- Send focused window to system tray:
+
+`kdocker -f`
+
+- Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button
```

Comment:
```suggestion
- Display a cursor to send a window to system tray with a custom icon when pressing the left mouse button:
```

Reply:
You have a button to edit the file directly:
![Screenshot_20210524_183935](https://user-images.githubusercontent.com/1536740/119410562-6db0a280-bcbf-11eb-86bc-e1f82856bcaa.png)

Reply:
That's cool; I've never actually used that before!

Reply:
Fully aware, just quicker for to do it there, since I was already there :laughing: